jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/ResourceService.java

```diff
@@ -160,6 +213,15 @@ public void doGet(Request request, Response response, Callback callback, HttpCon
 
         try
         {
+            if (isStylesheetRequest(pathInContext))
```

Comment:
Is this the right place for this? Perhaps we should first look for the real resource and only check if it is a stylesheet request if it misses.  This has better performance for other resource hits and allows the stylesheet to be overriden is specific directories.